translate/storage/jsonl10n.py

```diff
@@ -416,6 +416,120 @@ def _extract_units(
             yield from parent
 
 
+class GoTextJsonUnit(BaseJsonUnit):
+    ID_FORMAT = "{}"
+
+    def __init__(self, comment=None, message=None, meaning=None, key=None,
+                 fuzzy=None, position=None, **kwargs):
+        super().__init__(**kwargs)
+        self.comment = comment
+        self.message = message
+        self.meaning = meaning
+        self.key = key
+        self.fuzzy = fuzzy
+        self.position = position
+
+    def getvalue(self):
+        target = self.target
+        if isinstance(target, multistring):
+            strings = list(target.strings)
+            if len(self._store.plural_tags) > len(target.strings):
+                strings += [""] * (len(self._store.plural_tags) - len(target.strings))
+            target = {
+                "select": {
+                    "feature": "plural",
+                }
+            }
+            if self.placeholders and len(self.placeholders) > 0:
```

Comment:
Checking for length is not needed, these conditions are equivalent for a list.

```suggestion
            if self.placeholders:
```